Template.php

```diff
@@ -90,6 +90,10 @@ public function process() {
         $this->use_unnamed_params();
         $this->get_identifiers_from_url();
         $this->tidy();
+        if (strpos($this->get('url') , 'google.com') && $this->expand_by_google_books()) {
```

Comment:
Can we be more precise in our URL search, i.e. looking for a component that shows us that we're at google books not google finance (etc)?

Reply:
Done.